src/guardian/guardian.service.ts

```diff
@@ -96,10 +97,8 @@ export class GuardianService implements OnModuleInit {
     this.logger.log('New staking router state cycle start');
 
     try {
-      const {
-        elBlockSnapshot: { blockHash, blockNumber },
-        data: stakingModules,
-      } = await this.stakingRouterService.getStakingModules();
+      const { blockHash, blockNumber, stakingModulesData } =
+        await this.stakingRouterService.getVettedAndUnusedKeys();
```

Comment:
The result of this function is not a VettedAndUnusedKeys, but stakingModulesData

Reply:
fixed